src/test/java/com/uber/cadence/internal/compatibility/ProtoObjects.java

```diff
@@ -1324,6 +1324,65 @@ public final class ProtoObjects {
       UpdateDomainResponse.newBuilder().setDomain(DOMAIN).build();
 
+  public static final GetSearchAttributesRequest GET_SEARCH_ATTRIBUTES_REQUEST =
+      GetSearchAttributesRequest.getDefaultInstance();
+
+  public static final RegisterDomainResponse REGISTER_DOMAIN_RESPONSE =
+      RegisterDomainResponse.getDefaultInstance();
+
+  public static final DeprecateDomainResponse DEPRECATE_DOMAIN_RESPONSE =
+      DeprecateDomainResponse.getDefaultInstance();
+
+  public static final SignalWorkflowExecutionResponse SIGNAL_WORKFLOW_EXECUTION_RESPONSE =
+      SignalWorkflowExecutionResponse.getDefaultInstance();
+
+  public static final RequestCancelWorkflowExecutionResponse
+      REQUEST_CANCEL_WORKFLOW_EXECUTION_RESPONSE =
+          RequestCancelWorkflowExecutionResponse.getDefaultInstance();
+
+  public static final TerminateWorkflowExecutionResponse TERMINATE_WORKFLOW_EXECUTION_RESPONSE =
+      TerminateWorkflowExecutionResponse.getDefaultInstance();
+
+  public static final GetClusterInfoRequest GET_CLUSTER_INFO_REQUEST =
+      GetClusterInfoRequest.getDefaultInstance();
+
+  public static final RespondDecisionTaskFailedResponse RESPOND_DECISION_TASK_FAILED_RESPONSE =
+      RespondDecisionTaskFailedResponse.getDefaultInstance();
+
+  public static final RespondActivityTaskCompletedResponse
+      RESPOND_ACTIVITY_TASK_COMPLETED_RESPONSE =
+          RespondActivityTaskCompletedResponse.getDefaultInstance();
+
+  public static final RespondActivityTaskCompletedByIDResponse
+      RESPOND_ACTIVITY_TASK_COMPLETED_BY_ID_RESPONSE =
+          RespondActivityTaskCompletedByIDResponse.getDefaultInstance();
+
+  public static final RespondActivityTaskFailedResponse RESPOND_ACTIVITY_TASK_FAILED_RESPONSE =
+      RespondActivityTaskFailedResponse.getDefaultInstance();
+
+  public static final RespondActivityTaskFailedByIDResponse
+      RESPOND_ACTIVITY_TASK_FAILED_BY_ID_RESPONSE =
+          RespondActivityTaskFailedByIDResponse.getDefaultInstance();
+
+  public static final RespondActivityTaskCanceledResponse RESPOND_ACTIVITY_TASK_CANCELED_RESPONSE =
+      RespondActivityTaskCanceledResponse.getDefaultInstance();
+
+  public static final RespondActivityTaskCanceledByIDResponse
+      RESPOND_ACTIVITY_TASK_CANCELED_BY_ID_RESPONSE =
+          RespondActivityTaskCanceledByIDResponse.getDefaultInstance();
+
+  public static final RespondQueryTaskCompletedResponse RESPOND_QUERY_TASK_COMPLETED_RESPONSE =
+      RespondQueryTaskCompletedResponse.getDefaultInstance();
+
+  public static final ResetStickyTaskListResponse RESET_STICKY_TASK_LIST_RESPONSE =
+      ResetStickyTaskListResponse.getDefaultInstance();
+
+  public static final RefreshWorkflowTasksRequest REFRESH_WORKFLOW_TASKS_REQUEST =
+      RefreshWorkflowTasksRequest.getDefaultInstance();
+
+  public static final RefreshWorkflowTasksResponse REFRESH_WORKFLOW_TASKS_RESPONSE =
+      RefreshWorkflowTasksResponse.getDefaultInstance();
+
```

Comment:
I'm not 100% sure how much we care about these, but if we do care about them, is there some kind of assertion we can add to ensure that they're mapped for all future fields too? (maybe reflection?)

Reply:
For any type that we have a mapper for we use [MapperTestUtil](https://github.com/uber/cadence-java-client/blob/master/src/test/java/com/uber/cadence/internal/compatibility/MapperTestUtil.java) to assert that all fields are set (or a select set are missing if we haven't updated the mappers for them). We do similar things with some of the types contained within the requests(decisions and history events) as well as enums.

These types don't actually exist in thrift (the thrift APIs return void, something we never do in gRPC) so there aren't any mappers. It seems very unlikely we'd add fields to these types until after we completely remove thrift, at which point this will Thrift2Proto layer won't exist at all. So I don't think it's worth testing these ones the way all the others are tested.